lib/generators/rspec/install/templates/spec/rails_helper.rb

```diff
@@ -34,7 +34,13 @@
 RSpec.configure do |config|
 <% if RSpec::Rails::FeatureCheck.has_active_record? -%>
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
+<% if ::Rails::VERSION::STRING < "7.1.0" -%>
   config.fixture_path = Rails.root.join('spec/fixtures')
```

Comment:
```suggestion
  config.fixture_path = Rails.root.join('spec/fixtures')
  # Starting with Rails 7.1, use this instead:
  # config.fixture_paths = [
  #  Rails.root.join('spec/fixtures')
  # ]
```

Reply:
Would you default to a deprecated method for a brand new installation? Is it to keep the template simple?
We'll need to change the file again once Rails 7.2 is released in that case.

Reply:
My suggestion is to add a note and a clear suggestion on how to update for those starting with Rails 7.0 which is the current one.
When Rails 7.1 becomes current, the one from the below `else` clause with `fixture_paths = ` will be added.

Reply:
Oh, so sorry! I misread your suggestion. Makes sense